src/App.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import '../src/css/style.css';
-import Navbar from './pages/Navbar';
+import '../src/css/responsive.css';
+import Navbar from './components/Navbar';
+import CreateAccount from './components/CreateAccount';
 import LoginPage from './components/LoginPage';
+import ProfilePage from './pages/ProfilePage/ProfilePage';
 import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 
+
 function App() {
     return (
         <Router>
           {/* <Navbar /> */}
           <Routes>
-            <Route exact path="/userlogin" element={<LoginPage />} />
+            <Route exact path="/" element={<LoginPage />} />
+            <Route exact path="signup" element={<CreateAccount />} />
+            <Route exact path="/login" element={<LoginPage />} />
+            <Route exact path="/profile" element={<ProfilePage />} />
```

Comment:
Is profile page route in authorized or unauthorized

Reply:
Profile page route will be authorized. If user is successfully logged in, only then he/she will be redirected to the his/her profile page.